app/src/features/editor/components/ToolchainDropdown.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import Tooltip from '@mui/material/Tooltip';
+import Select from '@mui/material/Select';
+import MenuItem from '@mui/material/MenuItem';
+import FormControl from '@mui/material/FormControl';
+
+const ToolchainNames = [
+  'latest',
+  'nightly',
```

Comment:
Not sure if we should allow users to select the nightly option, how do you feel about just using `latest` along with the beta options?

Reply:
We could, but why would we disallow nightly?

Reply:
I was just thinking nightly might be a little too bleeding edge for the users but it's probably good to have it as an option. Can we have it show at the bottom of the list? Seems to be the rust playground approach. 

![Screenshot 2023-08-15 at 9 24 15 am](https://github.com/FuelLabs/sway-playground/assets/1289413/37df0231-6ea9-4d37-8e3e-f0cb9e698949)

Reply:
Done, I put nightly and latest at the bottom.